app/Services/FiscalYearService.php

```
@@ -0,0 +1,83 @@
+                    'period_start' => $lastOpenFiscalYear->period_start->toDateString(),
+                    'period_end' => $lastOpenFiscalYear->period_end->toDateString(),
+                    'reason' => 'Creating new fiscal year',
+                    'executed_by' => 'console',
+                    'timestamp' => now(),
+                ]);
+            }
+            $fiscalYear = FiscalYear::create([
+                'period_start' => $nextYear->startOfYear(),
+                'period_end' => $nextYear->endOfYear(),
+                'status' => PostingPeriodStatusType::OPEN,
+            ]);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\FiscalYear' in method 'createNextYearFiscalYear'. (undefined)

(StaticAccess)